lib/github-pages-health-check/domain.rb

```diff
@@ -78,10 +78,10 @@ class Domain < Checkable
       ).freeze
 
       CURRENT_IPV6_ADDRESSES = %w(
-        2606:50c0:8000::1538
-        2606:50c0:8001::1538
-        2606:50c0:8002::1538
-        2606:50c0:8003::1538
+        2606:50c0:8000::153
+        2606:50c0:8001::153
+        2606:50c0:8002::153
+        2606:50c0:8003::153
```

Comment:
👍 , but just for confirmation:

```sh
curl https://api.github.com/meta | jq .pages | grep ::

  "2606:50c0:8000::153/128",
  "2606:50c0:8001::153/128",
  "2606:50c0:8002::153/128",
  "2606:50c0:8003::153/128"
```

Reply:
Thanks. Yeah I bet I had a bad block cut on the CIDR and left the `8` 🙈